lib/src/json_schema/validator.dart

```diff
@@ -63,6 +63,25 @@ class Instance {
   toString() => data.toString();
 }
 
+/// The result of validating data against a schema
+class ValidationResults {
```

Comment:
I wonder if this should contain a simple isValid too? That'll make all your `.errors.isEmpty` go away too

Reply:
Yeah I wondered about that. It's redundant, but if it's nicer to use, we can add it.